redirects.yaml

```diff
@@ -1,4 +1,4 @@
-(?P<version>2\.3|2\.4|2\.5)(?P<path>.*): https://old-docs.jujucharms.com/{version}{path}
+(?P<version>1\.[0-9]{1,2}|2\.0|2\.1|2\.2|2\.3|2\.4|2\.5)(?P<path>.*): https://old-docs.jujucharms.com/{version}{path}
```

Comment:
It would be really nice if complex regex's had comments so that people dont' have to read them too close to grok it.

Reply:
I would agree with you if this regex were complicated... :p

Seriously though, I think adding comments to a python regex inside a yaml key while keeping it as valid yaml is gonna be pretty tricky. I'm not gonna attemt it right now.